src/main/windows/main.js

```diff
@@ -136,7 +136,11 @@ function setProgress (progress) {
 
 function setTitle (title) {
   if (!main.win) return
-  main.win.setTitle(title)
+  if (title) {
+    main.win.setTitle(`${config.APP_WINDOW_TITLE} - ${title}`)
+  } else {
+    main.win.setTitle(config.APP_WINDOW_TITLE)
+  }
```

Comment:
We force the title to always use the `APP_WINDOW_TITLE` with an optional email.

Reply:
Maybe should rename `title` to `email`.

Reply:
at some point we might want to add other kinds of titles here so i think the abstraction is a good one